src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -174,8 +174,10 @@ var DOMPropertyOperations = {
         var propName = propertyInfo.propertyName;
         // Must explicitly cast values for HAS_SIDE_EFFECTS-properties to the
         // property type before comparing; only `value` does and is string.
+        // Must set `value` property if it is not null and not yet set.
```

Comment:
Not saying it shouldn't be changed, but it only applies to `HAS_SIDE_EFFECTS` which is only used by `value`.